src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card-img">
+        <img 
+          class="img"
+          src="images/imac.jpeg"
+          alt="#"
+        >
+      </div>
+        
+      <h2 class="title">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h2>
+      <h3 class="subtitle">Product code: 195434</h3>
+        
+      <div class="card-reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="reviews">
+          <p>Reviews: 5</p>
+        </div>
+      </div>
+      <div class="card-price">
+        <div class="card-price__left">Price:</div>
+        <div class="card-price__right">$2,199</div>
+      </div>
+      
+      <a 
+        class="card-link"
+        href="#"
+        data-qa="hover"
+        >
+        buy
+      </a>
```

Comment:
1) as a card element
2) apply code styling recommendations from checklist
```suggestion
      <a 
        class="card__link"
        href="#"
        data-qa="hover"
      >
        buy
      </a>
```